src/routes/boot.ts

```diff
@@ -90,6 +90,7 @@ export type LoggedInBoot = BaseBoot & {
     canSubmitArticle: boolean;
   };
   accessToken: AccessToken;
+  marketingCta: null;
```

Comment:
This should be of type `marketingCta?: MarketingCta` right? Same below.

Reply:
It should, just haven't been implemented yet. Just added it to "reserve" the name for it. Shouldn't be an issue for now if I just merge it into a epic branch

Reply:
Yes, that works 👍